README.md

```diff
@@ -4,32 +4,45 @@
 
 ## Table of Contents
 
-- [About](#about)
-- [Prerequisites](#prerequisites)
-- [Quick Start](#quick-start)
-  * [Preparing your GPU Nodes](#preparing-your-gpu-nodes)
-  * [Enabling GPU Support in Kubernetes](#enabling-gpu-support-in-kubernetes)
-  * [Running GPU Jobs](#running-gpu-jobs)
-- [Configuring the NVIDIA device plugin binary](#configuring-the-nvidia-device-plugin-binary)
-  * [As command line flags or envvars](#as-command-line-flags-or-envvars)
-  * [As a configuration file](#as-a-configuration-file)
-  * [Configuration Option Details](#configuration-option-details)
-  * [Shared Access to GPUs](#shared-access-to-gpus)
-    * [With CUDA Time-Slicing](#with-cuda-time-slicing)
-    * [With CUDA MPS](#with-cuda-mps)
-- [Deployment via `helm`](#deployment-via-helm)
-  * [Configuring the device plugin's `helm` chart](#configuring-the-device-plugins-helm-chart)
-    + [Passing configuration to the plugin via a `ConfigMap`.](#passing-configuration-to-the-plugin-via-a-configmap)
-      - [Single Config File Example](#single-config-file-example)
-      - [Multiple Config File Example](#multiple-config-file-example)
-      - [Updating Per-Node Configuration With a Node Label](#updating-per-node-configuration-with-a-node-label)
-    + [Setting other helm chart values](#setting-other-helm-chart-values)
-    + [Deploying with gpu-feature-discovery for automatic node labels](#deploying-with-gpu-feature-discovery-for-automatic-node-labels)
-    + [Deploying gpu-feature-discovery in standalone mode](#deploying-gpu-feature-discovery-in-standalone-mode)
-  * [Deploying via `helm install` with a direct URL to the `helm` package](#deploying-via-helm-install-with-a-direct-url-to-the-helm-package)
-- [Building and Running Locally](#building-and-running-locally)
-- [Changelog](#changelog)
-- [Issues and Contributing](#issues-and-contributing)
+- [NVIDIA device plugin for Kubernetes](#nvidia-device-plugin-for-kubernetes)
```

Comment:
Question: What is the intent of this top-level element? It doesn't really seem to add value from a text / navigation perspective.

Reply:
It was some auto-format work that snuck in inadvertently. Removed.